skbio/diversity/_driver.py

```diff
@@ -207,6 +209,11 @@ def beta_diversity(metric, counts, ids=None, validate=True, **kwargs):
         bypassed if you're not certain that your input data are valid. See
         :mod:`skbio.diversity` for the description of what validation entails
         so you can determine if you can safely disable validation.
+    pairwise_func : callable, optional
```

Comment:
Also note what it should return (so that you've completely described the function signature)